data/styles/clear/include/Basemap.mapcss

```diff
@@ -98,14 +98,15 @@ area|z12-[natural=grassland],
 area|z12-[leisure=golf_course],
 area|z12-[natural=heath],
 area|z12-[landuse=allotments],
-area|z14-[natural=scrub],
 area|z12-[natural=bare_rock],
 area|z12-[landuse=orchard],
 area|z12-[landuse=vineyard],
 area|z12-[landuse=meadow],
 area|z12-[landuse=recreation_ground],
 area|z12-[landuse=village_green],
 area|z12-[landuse=field],
+area|z14-[natural=scrub],
+area|z16-[landuse=flowerbed],
```

Comment:
maybe you can also restore scrubs to z12- if you feel like it?
they were demoted by mistake https://github.com/organicmaps/organicmaps/pull/3786#issuecomment-1467947754

Reply:
will do :+1: